src/main/groovy/org/labkey/gradle/task/RunTestSuite.groovy

```diff
@@ -33,6 +34,7 @@ class RunTestSuite extends RunUiTest
         project.logger.info("RunTestSuite: constructor");
         scanForTestClasses = false
         include "org/labkey/test/Runner.class"
+        // TODO: Remove when 19.2 is no longer supported. Dependency is now declared in `qc/build.gradle`
```

Comment:
Shouldn't this say 19.1?  We're targeting this change for 19.2.

Reply:
Wasn't sure about that. If we claim to support a particular version of LabKey, does that mean any pre-release/alpha version or just the final release?.

Reply:
Just the final release.  Pre-releases are inherently unstable in my opinion.